Gems/GraphCanvas/Code/Source/GraphCanvas.cpp

```diff
@@ -385,34 +384,6 @@ namespace GraphCanvas
         AZ::Data::AssetCatalogRequestBus::Broadcast(&AZ::Data::AssetCatalogRequestBus::Events::EnumerateAssets, nullptr, collectAssetsCb, postEnumerateCb);
     }
 
-    void GraphCanvasSystemComponent::RegisterAssetHandler()
-    {
-        AZ::Data::AssetType assetType(azrtti_typeid<TranslationAsset>());
-        if (AZ::Data::AssetManager::Instance().GetHandler(assetType))
-        {
-            return; // Asset Type already handled
-        }
-
-        auto* catalogBus = AZ::Data::AssetCatalogRequestBus::FindFirstHandler();
-        if (catalogBus)
-        {
-            // Register asset types the asset DB should query our catalog for.
-            catalogBus->AddAssetType(assetType);
-
-            // Build the catalog (scan).
-            catalogBus->AddExtension(".names");
-        }
-
-        m_assetHandler = AZStd::make_unique<TranslationAssetHandler>();
-        AZ::Data::AssetManager::Instance().RegisterHandler(m_assetHandler.get(), assetType);
-
-        // Use AssetCatalog service to register ScriptEvent asset type and extension
-        AZ::Data::AssetCatalogRequestBus::Broadcast(&AZ::Data::AssetCatalogRequests::AddAssetType, assetType);
-        AZ::Data::AssetCatalogRequestBus::Broadcast(&AZ::Data::AssetCatalogRequests::EnableCatalogForAsset, assetType);
-        AZ::Data::AssetCatalogRequestBus::Broadcast(&AZ::Data::AssetCatalogRequests::AddExtension, TranslationAsset::GetFileFilter());
-
-    }
-
```

Comment:
This shouldn't be removed, it will break the JSON based text replacement feature. Nothing is using it yet so this shouldn't be causing problems; if it is, let's discuss what they are and how to fix them without cutting the feature.